src/sinol_make/commands/inwer/__init__.py

```diff
@@ -0,0 +1,115 @@
+import argparse
+import os
+import multiprocessing as mp
+
+from sinol_make import util
+from sinol_make.commands.inwer.structs import TestResult, InwerExecution, VerificationResult, TableData
+from sinol_make.helpers import package_util
+from sinol_make.helpers.parsers import add_compilation_arguments
+from sinol_make.interfaces.BaseCommand import BaseCommand
+from sinol_make.commands.inwer import inwer_util
+
+
+class Command(BaseCommand):
+    """
+    Class for "inwer" command.
+    """
+
+    def get_name(self):
+        return "inwer"
+
+    def configure_subparser(self, subparser: argparse.ArgumentParser):
+        parser = subparser.add_parser(
+            self.get_name(),
+            help='Verify if input files are correct',
+            description='Verify if input files are correct'
+        )
+
+        parser.add_argument('inwer', type=str, nargs='?',
+                            help='path to inwer executable')
+        parser.add_argument('--tests', type=str, nargs='+',
+                            help='test to verify, for example in/abc{0,1}*')
+        parser.add_argument('--cpus', type=int,
+                            help=f'number of cpus to use, you have {mp.cpu_count()} available')
+        add_compilation_arguments(parser)
+
+    def compile_inwer(self, args: argparse.Namespace):
+        self.inwer_executable, compile_log = inwer_util.compile_inwer(self.inwer, args)
+        if self.inwer_executable is None:
+            print(util.error('Compilation failed.'))
+            os.system("head -c 500 %s" % compile_log)
+            exit(1)
+        else:
+            print(util.info('Compilation successful.'))
+
+    def verify_test(self, execution: InwerExecution) -> VerificationResult:
+        """
+        Veryfies test and returns results.
+        """
+        output_dir = os.path.join(os.getcwd(), 'cache', 'executions', execution.test_name)
+        os.makedirs(output_dir, exist_ok=True)
+        output_file = os.path.join(output_dir, 'inwer.out')
+
+        command = f'{execution.inwer_exe_path} {execution.test_name} < {execution.test_path} > {output_file}'
+        exit_code = os.system(command)
+
+        return VerificationResult(
+            execution.test_path,
+            exit_code == 0,
+            open(output_file, 'r').read()
+        )
```

Comment:
There might be some built-in Python functions to execute a command and capture it's output and return code. Using it would shorten the code and reduce the chances that there is a bug.

Reply:
I don't think there is a better way of running shell scripts than `os.system` or `subprocess.Popen` (which in this case would look exactly the same).

Reply:
I changed the `inwer` executable name, so now the `inwer` command can be run in parallel: 7f667d24dce48ba6cde38b78869b46f65f3512df.